src/services

```diff
@@ -37,7 +37,7 @@ p({"class":"foam.nanos.boot.NSpec", "name":"classes",                          "
 p({"class":"foam.nanos.boot.NSpec", "name":"threads",                          "lazy":true,  "serve":false, "serviceClass":"foam.nanos.http.ThreadsWebAgent"})
 p({"class":"foam.nanos.boot.NSpec", "name":"login",                            "lazy":true,  "serve":false, "serviceClass":"foam.nanos.http.ShowSessionWebAgent"})
 p({"class":"foam.nanos.boot.NSpec", "name":"memory",                           "lazy":true,  "serve":false, "serviceScript":"return new foam.nanos.http.RefreshWebAgent(new foam.nanos.http.MemoryWebAgent());"})
-p({"class":"foam.nanos.boot.NSpec", "name":"countryDAO",                       "lazy":true,  "serve":true,  "serviceScript":"return new foam.dao.EasyDAO.Builder(x).setPm(true).setAuthenticateRead(false).setJournalType(foam.dao.JournalType.SINGLE_JOURNAL).setJournalName(\"countries\").setOf(foam.nanos.auth.Country.getOwnClassInfo()).setOrder(new foam.core.PropertyInfo[] {foam.nanos.auth.Country.NAME}).build();","client":"{\"of\":\"foam.nanos.auth.Country\"}"})
+p({"class":"foam.nanos.boot.NSpec","name":"countryDAO","description":"A DAO which contains information about countries.","serve":true,"authenticate":false,"serviceScript":"return new foam.dao.EasyDAO.Builder(x)\u000a  .setPm(true)\u000a  .setAuthenticateRead(false)\u000a  .setJournalType(foam.dao.JournalType.SINGLE_JOURNAL)\u000a  .setJournalName(\"countries\")\u000a  .setOf(foam.nanos.auth.Country.getOwnClassInfo())\u000a  .setOrder(new foam.core.PropertyInfo[] {foam.nanos.auth.Country.NAME})\u000a  .build();","client":"{\"of\":\"foam.nanos.auth.Country\"}","authNotes":"The \"authenticate\" flag is explicitly set to false because we want to be able to use the countryDAO to populate a dropdown in the sign up form."})
```

Comment:
support multi-line scripts now thanks to @michaelconnor468 

You can write this as
p({
  "class": "foam.nanos.boot.NSpec", "name": ...
  "serviceScript": """
return new foam.dao.EasyDAO.Builder(x).
  setFoo().
  setBar().
...
  build();
""", "other properties"
})
```

A multi-line entry is delimited by a leading "p({" on its own line and terminated by "})" on its own line.

Reply:
Resolved by 4cf4c0b.